<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Erfassen</class>
 <widget class="QDialog" name="Erfassen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaktion erfassen</string>
  </property>
  <widget class="QPushButton" name="btn_ErErfassen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Erfassen</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Betrag:*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum:*</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="txt_erf_date">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_erf_amount">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kategorie:*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_cat">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_komma">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_decimal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtn_einnahme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Einnahme</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtn_ausgabe">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ausgabe</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quelle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zahlungsart:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_erf_source">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_paymentOption">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>161</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Beschreibung:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_erf_description">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ErAbbrechen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_erf_date</tabstop>
  <tabstop>txt_erf_amount</tabstop>
  <tabstop>txt_decimal</tabstop>
  <tabstop>combo_cat</tabstop>
  <tabstop>txt_erf_source</tabstop>
  <tabstop>combo_paymentOption</tabstop>
  <tabstop>txt_erf_description</tabstop>
  <tabstop>btn_ErErfassen</tabstop>
  <tabstop>btn_ErAbbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
